### PR TITLE
PREFIXING: remove vendor prefixes

### PR DESCRIPTION
**Pull request by estelle**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/glossary/vendor_prefix/index.md [hunk 1]
```diff
@@ -5,20 +5,20 @@ tags:
...
 
-> **Note:** Browser vendors are working to stop using vendor prefixes for experimental features. Web developers have been using them on production Web sites, despite their experimental nature. This has made it more difficult for browser vendors to ensure compatibility and to work on new features; it's also been harmful to smaller browsers who wind up forced to add other browsers' prefixes in order to load popular web sites.
->
-> Lately, the trend is to add experimental features behind user-controlled flags or preferences, and to create smaller specifications which can reach a stable state much more quickly.
+Web developers included prefixed features on production Web sites, despite their experimental nature. This made it more difficult for browser vendors to ensure compatibility while working on new features. Including prefixed features also harmed smaller browsers who ended up having to add other browsers' prefixes in order to load popular web sites.
```
Comment (teoli2003) [inline files/en-us/glossary/vendor_prefix/index.md:10]:
```suggestion
Web developers included prefixed features on production websites, despite their experimental nature. This made it more difficult for browser vendors to ensure compatibility while working on new features. Including prefixed features also harmed smaller browsers who ended up having to add other browsers' prefixes to load popular websites.
```

File: files/en-us/glossary/vendor_prefix/index.md [hunk 2]
```diff
@@ -62,6 +68,12 @@ window.requestAnimationFrame = window.requestAnimationFrame ||
                                window.msRequestAnimationFrame;
 ```
 
+If you encounter the above code in a code base, you can safely remove all but the first line. All browsers support [`requestAnimationFrame`](/en-US/docs/Web/API/window/requestAnimationFrame#browser_compatibility) without vendor prefixes:
+
+```js
+window.requestAnimationFrame = window.requestAnimationFrame;
```
Comment (teoli2003) [inline files/en-us/glossary/vendor_prefix/index.md:74]: No need to declare it, not even a need to have `window.` in front of it.
```suggestion
requestAnimationFrame(callback);
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/glossary/vendor_prefix/index.md [hunk 3]
```diff
@@ -30,27 +30,33 @@ Sample usage:
...
+If you encounter the above code in a code base, you can safely remove all but the last line. All browsers support [transitions](/en-US/docs/Web/CSS/transition#browser_compatibility) without vendor prefixes:
+
+```css
+transition: all 4s ease;
+```
+
 ## API prefixes
 
-Historically, vendors have also used prefixes for experimental APIs. If an entire interface is experimental, then the interface's name is prefixed (but not the properties or methods within). If an experimental property or method is added to a standardized interface, then the individual method or property is prefixed.
+Historically, vendors have also used prefixes for experimental APIs. If an entire interface was experimental, then the interface's name was prefixed (but not the properties or methods within). If an experimental property or method was added to a standardized interface, then the individual method or property was prefixed.
```
Comment (dipikabh) [inline files/en-us/glossary/vendor_prefix/index.md:41]:
```suggestion
Historically, browser vendors have also used prefixes for experimental APIs. If an entire interface was experimental, then the interface's name was prefixed (but not the properties or methods within). If an experimental property or method was added to a standardized interface, then the individual method or property was prefixed.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for updating this information, @estelle!

File: files/en-us/glossary/vendor_prefix/index.md [hunk 4]
```diff
@@ -62,6 +68,12 @@ window.requestAnimationFrame = window.requestAnimationFrame ||
...
 
+If you encounter the above code in a code base, you can safely remove all but the first line. All browsers support [`requestAnimationFrame`](/en-US/docs/Web/API/window/requestAnimationFrame#browser_compatibility) without vendor prefixes:
+
+```js
+window.requestAnimationFrame = window.requestAnimationFrame;
+```
+
 ## See also
 
 - [Vendor prefix](https://en.wikipedia.org/wiki/CSS_hack#Browser_prefixes) on Wikipedia
```
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/glossary/vendor_prefix/index.md:79]: Should we rename this page?

File: files/en-us/glossary/vendor_prefix/index.md [hunk 5]
```diff
@@ -62,6 +68,12 @@ window.requestAnimationFrame = window.requestAnimationFrame ||
                                window.msRequestAnimationFrame;
 ```
 
+If you encounter the above code in a code base, you can safely remove all but the first line. All browsers support [`requestAnimationFrame`](/en-US/docs/Web/API/window/requestAnimationFrame#browser_compatibility) without vendor prefixes:
```
Comment (estelle) [inline files/en-us/glossary/vendor_prefix/index.md:71]:
```suggestion
If you encounter the above code in a code base, you can safely remove all but the first line. All browsers support [`requestAnimationFrame`](/en-US/docs/Web/API/window/requestAnimationFrame#browser_compatibility) without vendor prefixes, and without `window`:
```

**Review by teoli2003 — APPROVED** [review]
LGTM

File: files/en-us/glossary/vendor_prefix/index.md [hunk 6]
```diff
@@ -5,20 +5,20 @@ tags:
...
 
-> **Note:** Browser vendors are working to stop using vendor prefixes for experimental features. Web developers have been using them on production Web sites, despite their experimental nature. This has made it more difficult for browser vendors to ensure compatibility and to work on new features; it's also been harmful to smaller browsers who wind up forced to add other browsers' prefixes in order to load popular web sites.
->
-> Lately, the trend is to add experimental features behind user-controlled flags or preferences, and to create smaller specifications which can reach a stable state much more quickly.
+Web developers included prefixed features on production websites, despite their experimental nature. This made it more difficult for browser vendors to ensure compatibility while working on new features. Including prefixed features also harmed smaller browsers who ended up having to add other browsers' prefixes to load popular websites.
```
Comment (estelle) [inline files/en-us/glossary/vendor_prefix/index.md:10]:
```suggestion
Web developers included prefixed features on production websites, despite their experimental nature. This made it more difficult for browser vendors to ensure compatibility while working on new features. Including prefixed features also harmed smaller browser vendors who ended up having to add other browsers' prefixes to load popular websites.
```

File: files/en-us/glossary/vendor_prefix/index.md [hunk 7]
```diff
@@ -5,20 +5,20 @@ tags:
...
 
-> **Note:** Browser vendors are working to stop using vendor prefixes for experimental features. Web developers have been using them on production Web sites, despite their experimental nature. This has made it more difficult for browser vendors to ensure compatibility and to work on new features; it's also been harmful to smaller browsers who wind up forced to add other browsers' prefixes in order to load popular web sites.
->
-> Lately, the trend is to add experimental features behind user-controlled flags or preferences, and to create smaller specifications which can reach a stable state much more quickly.
+Web developers included prefixed features on production websites, despite their experimental nature. This made it more difficult for browser vendors to ensure compatibility while working on new features. Including prefixed features also harmed smaller browser vendors who ended up having to add other browsers' prefixes to load popular websites.
```
Comment (estelle) [inline files/en-us/glossary/vendor_prefix/index.md:10]:
```suggestion
Web developers included prefixed features on production websites, despite their experimental nature. This made it more difficult for browser vendors to ensure compatibility while working on new features. Including prefixed features also harmed smaller browser vendors who ended up having to add other browsers' prefixes in order to render popular websites.
```

**Review by dipikabh — APPROVED** [review]
Looks good, thank you!

**Pull request merged**